### PR TITLE
Code refactoring for message package

### LINKED ISSUE #74: Code refactoring for message package
## Feature Request

**Is your feature request related to a problem? Please describe.**  

**Describe the solution you'd like**  

Split the code into the separated directory followed by the feature of each function of the `message` package.

**Describe alternatives you've considered**  

**Additional context**

### PR DESCRIPTION
**Pull request by anthonyminyungi**
# Summary 

- It closes #74 

## How to Review? 

- Please refer to my self-review submitted as the review.

### CODE REVIEW

File: packages/message/src/index.ts [hunk 1]
```diff
@@ -1,67 +1,3 @@
...
-}): Property[] {
-  return Object.keys(object).map((key) => ({
-    type: PropertyType.Raw,
-    key,
-    value: object[key],
-  }));
-}
+export * from './type';
+export * from './util';
+export * from './property';
```
Comment (anthonyminyungi) [inline packages/message/src/index.ts:3]: I like this expression.

**Review by anthonyminyungi — COMMENTED** [review]
I split the functions and types in the file `index.ts`  into each directory by their characteristics, also each file.

File: packages/message/src/index.ts (hunk 1 above)
Comment (HyeockJinKim) [reply packages/message/src/index.ts:3]: cool!

File: packages/message/src/util/hash.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import sha256 from 'crypto-js/sha256';
+
+export function hash(msg: string): string {
+  return sha256(msg).toString();
+}
```
Comment (HyeockJinKim) [inline packages/message/src/util/hash.ts:5]: Please use the `utils` folder name.
And in each folder, export all functions from ʻindex.ts` in same directory (`utils/index.ts`).

**Review by HyeockJinKim — CHANGES_REQUESTED** [review]
Take a look my review.

File: packages/message/src/property/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export * from './hashProperty';
+export * from './objectToProperty';
+export * from './propertyObject';
```
Comment (anthonyminyungi) [inline packages/message/src/property/index.ts:3]: The exportation statement was changed.

File: packages/message/src/utils/hash.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import sha256 from 'crypto-js/sha256';
+
+export function hash(msg: string): string {
```
Comment (anthonyminyungi) [inline packages/message/src/utils/hash.ts:3]: The directory name was changed `util` -> `utils`.

**Review by anthonyminyungi — COMMENTED** [review]
Review reflected.

**Review by HyeockJinKim — APPROVED** [review]
LGTM

**Pull request merged**